TelCo.ColorCoder/ColorCoderCheck.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Drawing;
+using System.Text;
+
+namespace TelCo.ColorCoder
+{
+    class ColorCoderCheck 
```

Comment:
What is the responsibilities of class name  "ColorCoderCheck" when it holds Test Suites
String Construction and format process @Console.WriteLine  process repeated 
Main method holds multiple test cases

Reply:
Modified the class name and removed the repetition of Console.WriteLine into separate functions.